src/components/detail/detail.style.ts

```diff
@@ -15,7 +18,7 @@ const StyledDetail = styled.div`
   }
 `;
 
-const StyledDetailContent = styled.div`
+const StyledDetailContent = styled.div<IconProps>`
```

Comment:
What do you think about alternatively doing this? I feel the name `IconProps` may be a bit misleading since this styled component is a container rather than an icon.

```suggestion
const StyledDetailContent = styled.div<{ hasIcon: boolean }>`
```